web_external/js/views/widgets/ImageViewerWidget.js

```diff
@@ -1,6 +1,18 @@
 isic.views.ImageViewerWidget = isic.View.extend({
     initialize: function (settings) {
         /* "model" and "el" must be passed as settings */
+
+        this.renderedModelId = null;
+
+        this.listenTo(this.model, 'change', this.fetchTileInfo);
```

Comment:
Note that if you instantiate a `ImageViewerWidget` with an `el` and `model`, then the model gets changed, the `ImageViewerWidget` instance will spontaneously render itself, even if `render()` was not yet explicitly called by the owner. I don't think there's an easy way around this though, and it's probably not going to be an issue, given the ways we're using `ImageViewerWidget` right now.